.gitignore

```diff
@@ -36,3 +36,8 @@
 .Trashes
```

Comment:
Instead of a folder for each page, the idea to split various page elements into various subfolders under client. 

client/sender/html/sender_index.html should go to client/html/
client/sender/js/*.js should go to client/js/ etc

Can we pls do that?